src/codecs/dds.rs

```diff
@@ -176,6 +204,25 @@ impl<R: Read> DdsDecoder<R> {
                 b"DXT1" => DxtVariant::DXT1,
                 b"DXT3" => DxtVariant::DXT3,
                 b"DXT5" => DxtVariant::DXT5,
+                b"DX10" => {
+                    let dx10_header = DX10Header::from_reader(&mut r)?;
+                    match dx10_header.dxgi_format {
+                        70 | 71 | 72 => DxtVariant::DXT1,
+                        73 | 74 | 75 => DxtVariant::DXT3,
+                        76 | 77 | 78 => DxtVariant::DXT5,
```

Comment:
Can you insert comments (with links to documentation) regarding these constants?  A link to 
<https://docs.microsoft.com/en-us/windows/win32/api/dxgiformat/ne-dxgiformat-dxgi_format>
and the general information that these refer to subvariants with different sample interpretation:
 typeless, unorm, unorm_srgb

Reply:
Links from the PR comments would also be good for this purpose! Those comments should belong in the commit message or code, not merely github's issue system.